### PR TITLE
Restaurantproduct

### PR DESCRIPTION
**Pull request by sidd-shah**

### CODE REVIEW

**Comment by tanmay-gp** [comment]
@sidd-shah Shouldn't the PR be on `feature_add_libproject_psts` branch?

**Comment by sidd-shah** [comment]
@tanmay-gp91 Nope, i wanted you to create feature_add_libproject_psts  on /Flutterbee/restaurantproduct

File: library/src/com/astuetz/PagerSlidingTabStrip.java [hunk 1]
```diff
@@ -33,593 +34,618 @@
...
+    private void addTextTab(LinearLayout tabLayout, String title) {
+
+        TextView tab = new TextView(getContext());
+        tab.setText(title);
+        tab.setGravity(Gravity.CENTER);
+        tab.setSingleLine();
+        tabLayout.addView(tab);
+    }
+
+    private void addCounterTab(LinearLayout tabLayout, String counter) {
```
Comment (vmalusare) [inline library/src/com/astuetz/PagerSlidingTabStrip.java:259]: Comment needed : what is this counter tab,where it will be displayed ??

Comment (tanmay-gp) [inline library/src/com/astuetz/PagerSlidingTabStrip.java:259]: @sidd-shah add the necessary comments

Comment (sidd-shah) [inline library/src/com/astuetz/PagerSlidingTabStrip.java:259]: done

**Pull request merged**